finder-app/finder-test.sh

```diff
@@ -22,6 +22,7 @@ then
 else
 	NUMFILES=$1
 	WRITESTR=$2
+	WRITEDIR=/tmp/$3
```

Comment:
@lnxblog using `/tmp/aeld-data` or similar might be good here so students don't have a bunch of random directories in `/tmp` to cleanup.  Also automatically deleting after a successful run would be good.

Reply:
@dwalkes The third argument is provided by assignment-test.sh for creating directory with random name. https://github.com/cu-ecen-aeld/assignment-autotest/pull/21
Any suggestions?

Reply:
@lnxblog I meant to use `/tmp/aeld-data/$3` so `rm -rf /tmp/aeld-data` will clean up all including any random directories